get_coverage_for_challenge.sh

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env bash
+
+set -x
+set -e
+set -u
+set -o pipefail
+
+SCRIPT_CURRENT_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+
+CHALLENGE_ID=$1
+NODEJS_TEST_REPORT_JSON_FILE="${SCRIPT_CURRENT_DIR}/coverage/coverage-summary.json"
+NODEJS_CODE_COVERAGE_INFO="${SCRIPT_CURRENT_DIR}/coverage.tdl"
+
+( cd ${SCRIPT_CURRENT_DIR} && npm install && npm run coverage || true 1>&2 )
+
+[ -e ${NODEJS_CODE_COVERAGE_INFO} ] && rm ${NODEJS_CODE_COVERAGE_INFO}
+
+if [ -f "${NODEJS_TEST_REPORT_JSON_FILE}" ]; then
+    prettier --write ${NODEJS_TEST_REPORT_JSON_FILE}
```

Comment:
This command might not be installed as a shell, it is better to reference it from `.node_modules` or find an alternative.

Reply:
I'll change it to run from node_modules

Reply:
Having checked the docker image, Im rnning:
```
RUN npm install -g prettier
```
Which deals with the presence of `prettier`, also the script works, i havent got an error so far, so we wont need to change.

Reply:
I need to also be able to run the script manually, so not relying on it being installed globally makes it easier.

Reply:
Ok i see that way, done that as well